src/CribblyBackend.DataAccess/Repositories/DivisionRepository.cs

```diff
@@ -35,6 +59,27 @@ public async Task<Division> Create(Division division)
 
             return division;
         }
+        public async Task<Division> AddTeam(int id, Team team)
+        {
+            if (this.GetById(id) != null)
+            {
+                var result = await _connection.ExecuteAsync(
+                    @"
+                        UPDATE Teams
+                        SET Teams.Division = @Id
+                        WHERE Teams.Id = @TeamId
+                    ",
+                    new {Id = id, TeamId = team.Id}
+                );
+
+                return await this.GetById(id);
+            }
+            else
+            {
+                throw new DivisionNotFoundException();
+            }
```

Comment:
the `else` and its block are unnecessary here; you can just `throw` after the `if`

Reply:
In fact, I think I'd prefer throwing first if the team doesn't exist, and then executing the UPDATE:
```
if (GetById(id) == null) 
{
    throw new DivisionNotFoundException();
}
// ... do the other stuff
```
This keeps our "happy path" more in the line of sight of the reader of the code

Reply:
fixed